model/PermissionProvider.php

```diff
@@ -136,4 +137,23 @@ public static function getSupportedRootClasses()
             new core_kernel_classes_Class(GenerisRdf::CLASS_ROLE)
         ];
     }
+
+    public function getResourceAccessData(string $resourceId): array
+    {
+        /** @var DataBaseAccess $db */
+        $db = $this->getServiceLocator()->get(DataBaseAccess::SERVICE_ID);
+        $results = $db->getUsersWithPermissions([$resourceId]);
+
+        $permissions = [];
+        foreach ($results as $result) {
+            $user = $result['user_id'];
+
+            if (!isset($permissions[$user])) {
+                $permissions[$user] = [];
+            }
```

Comment:
```suggestion
```

It's not a problem to create a new array entry and add a new element into it at the same time.